### DISCUSSION TITLE
GitHub thinks I am not near my school, but I am in my school to make the request.

### DISCUSSION THREAD

**Discussion by SpikeVN** [community/community#31898] [GitHub Education]
In my place, tracing geographic address from IP is terrible (the detected location is 100km away from where I am currently in). GitHub keep saying that I am not anywhere near my school campus despite the fact that I made that request in my school campus. Is there anything I can do about that?

**Comment by jfullstackdev** [comment]
is the school using any bypass or VPN ? because that will hide the true IP address, 

also, does the school have a static public IP address ? because it's hard to be traced without that
